app/Http/Controllers/StationController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Station;
+use App\Services\StationService;
+use App\Http\Requests\StoreStation;
+use App\Http\Requests\UpdateStation;
+use Illuminate\Http\Response;
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+
+class StationController extends Controller
+{
+    public function __construct(StationService $station)
+    {
+        $this->station = $station;
```

Comment:
This name is at least ambiguous. It's not a station, but service handling station, so variable should reflect that.

Reply:
Agreed. It was initially meant to be a repository with crud methods, so it made a bit more sense in this context, but as you can see below I'm still conflicted about the approach to database queries.